chapters/Index.md

```diff
@@ -35,24 +35,2394 @@ Jintek           <info@jintek.consulting>
+
+Dereferencing a DID URL that has no path, query, or fragment parts MUST return the DID document from resolution.
+
+BTCR2 provides no DID-method-specific dereferencing mechanisms. However, we anticipate a potential future option to enable a BTCR2 DID URL that dereferences Ordinal-based assets that are stored on-chain. However, this is out of scope for this specification.
+
+## Beacons
+
+### Overview
+
+A ::BTCR2 Beacon:: is a service listed in a BTCR2 DID document that informs resolvers how to find authentic updates to the DID. The service properties define a Bitcoin address to watch for transactions containing ::Beacon Signals::.
+
+All Beacon Signals broadcast from a ::BTCR2 Beacon:: in the ::Contemporary DID Document:: MUST be processed as part of DID document resolution. The ::Beacon Type:: in the service defines how ::Beacon Signals:: MUST be processed.
+
+Any on-chain Beacon Signal that cannot be processed renders the related DID invalid. For this reason, all Beacon Signals MUST ensure the approval by the DID controller, so that only approved Signals can be posted to Bitcoin. For resilience, BTCR2 DIDs can specify any number of Beacons and SHOULD include at least one Singleton Beacon as a fallback in case all Aggregation Beacons fail.
+
+A ::Beacon Signal:: commits to, and anchors in a Bitcoin block, 32 bytes of information that represents one of the following:
+
+* a ::BTCR2 Update Announcement::;  
+* the hash of a ::Beacon Announcement Map::; or  
+* the 32 bytes of an optimized ::Sparse Merkle Tree:: root, where each leaf node is deterministically selected by a **did:btcr2** identifier and contains a hash associated with the **did:btcr2** identifier.
+
+**did:btcr2** supports different ::Beacon Types::, with each type defining a set of algorithms for:
+
+1. How a ::BTCR2 Beacon:: can be established and added as a service to a DID document.  
+2. How ::BTCR2 Update Announcements:: are broadcast within ::Beacon Signals::.  
+3. How a resolver processes ::Beacon Signals::, identifying, verifying, and applying the authorized mutations to a DID document for a specific DID.
+
+This is an extensible mechanism, such that in the future new ::Beacon Types:: could be added.
+
+The current, active ::BTCR2 Beacons:: of a DID document are specified in the document's `service` property. By updating the DID document, a DID controller can change the set of ::BTCR2 Beacons:: they use to broadcast updates to their DID document over time. Resolution of a DID MUST process signals from all ::BTCR2 Beacons:: identified in the latest DID document and apply them in the order determined by the version specified by the `btcr2Update`.
+
+All resolvers of **did:btcr2** DIDs MUST support the ::Beacon Types:: defined in this specification.
+
+## Aggregation
```

Comment:
The heading heirarchy of this Beacon (and Aggregation) section feels wonky.

I would probably say aggregation should be a sub heading of Beacons.

I also wonder if Beacon Types should come first?